index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://use.fontawesome.com/cb342c03cb.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>T4G Research Group</title>
+</head>
+
+<body>
+    <header class="hero-image">
+        <div class="top-nav">
+            <ul>
+                <li id="logo"><a href="T4G-Logo.png">
+                    <img src="T4G-Logo.png" alt="T4G logo" width="65" height="65">
```

Comment:
Should these `width` and `height` attributes be moved out into the CSS file?

Reply:
CSS can be either in line on html page or in style sheet depending on how you want it to function.  The measurements were put directly on the image reference then added a width of 100% in CSS so they adjust with each devices width.   This is part of using the view-port option for device resizing.  One of many ways to skin the cat.  I'll look into it and see if there needs to be a modification.